common/scala/src/main/scala/whisk/core/entity/Parameter.scala

```diff
@@ -107,6 +124,11 @@ protected[entity] class ParameterName protected[entity] (val name: String) exten
  */
 protected[entity] class ParameterValue protected[entity] (private val value: JsValue) extends AnyVal {
     protected[core] def apply() = Option(value) getOrElse JsNull
+
+    /**
+     * The size of the ParameterValue entity as ObjectSize.
+     */
+    def size = value.toString.sizeInBytes
```

Comment:
As this is written into the database and sent over the network this is expected.